static/js/invite.js

```diff
@@ -120,6 +121,14 @@ function generate_multiuse_invite() {
                 }),
                 invite_status,
             );
+            $("#copy_link").click(() => {
+                const tempInput = document.createElement("input");
+                tempInput.value = data.invite_link;
+                document.body.append(tempInput);
```

Comment:
Why do we need to create a `tempInput` here?  Can't we simply select the text that is already in the widget?

Reply:
For copying the generated url, the text needs to be inside an input box ig, though the url is visible inside the widget, it is not present inside an input box.

Reply:
Steve, I've added the comments for better understanding

Reply:
Please put the comments on a line by themselves before the code they're describing.

Reply:
I'm not sure I believe that one needs an input box in order to copy text from the DOM -- don't we do this exact thing without this input hack elsewhere in the codebase?

(I suppose it might be just that we call a ClipboardJS function rather than doing it directly?)

@sumanthvrao FYI in case you can provide advice since you just added something similar elsewhere.

Reply:
Sure will go through that.

Reply:
@aniketsonu You should be able to use the `ClipboardJS` to do the copy here. There are multiple ways of doing this (https://clipboardjs.com/). It looks like using the `data-clipboard-target="#id"` variant might be the easiest here, but you should definitely experiment to see which is the simplest. You can also look at how we use `ClipboardJS` in other parts of our codebase (`git grep ClipboardJS`).

Reply:
@smuanthvrao okay sure will try that had a look at `data-clipboard-target` but the link is being generated like this
![Capture](https://user-images.githubusercontent.com/56171163/95752988-87682100-0cbe-11eb-8d0b-2b2dc5da7639.PNG)

Reply:
Sure! `i18n` is a part of our internationalization module (https://zulip.readthedocs.io/en/latest/translating/internationalization.html?highlight=i18n). This shouldn't matter for your issue.

Reply:
oh okay thanks!